en/docs/administer/logging-and-monitoring/logging/setting-up-logging.md

```diff
@@ -1,74 +1,105 @@
-# Setting up logging in API Manager
+# Setting up Logging in API Manager
 
 Logging is one of the most important aspects of a production-grade server. A properly configured logging system is vital for identifying errors, security threats, and usage patterns.
 
-WSO2 API Manager uses various types of logs to track realtime internal and external activities. Separate log files are created for each of those log types in the `<APIM_HOME>/repository/logs` directory. Following illustrates the log types supported by the APIM and how those logs can be configured.
+WSO2 API Manager uses various types of logs to track real-time internal and external activities. Separate log files are created for each of those log types in the `<APIM_HOME>/repository/logs` directory. The following illustrates the log types supported by the WSO2 API-M and how those logs can be configured.
 
-##Carbon Logs
+## Carbon Logs
 
-WSO2 API Manager is shipped with log4j2 logging capabilities which generate logs for administrative and server-side activities. By default, carbon logs are persisted in `wso2carbon.log` which is located in `<APIM_HOME>/repository/logs` directory. You can configure the details that are captured in this log file by configuring the `<APIM_HOME>/repository/conf/log4j2.properties`.
+WSO2 API Manager is shipped with the log4j2 logging capabilities that generate logs for administrative and server-side activities. By default, Carbon Logs are persisted in the `wso2carbon.log` file, which is located in thee `<APIM_HOME>/repository/logs` directory. You can configure the details that are captured in this log file by configuring the `<APIM_HOME>/repository/conf/log4j2.properties` file.
 
-!!!Java logging and Log4j integration
+!!! text "Java Logging and Log4j integration"
     In addition to the logs from libraries that use Log4j, all logs from libraries such as, Tomcat and Hazelcast that use Java logging framework are also visible in the same log files. That is, when Java logging is enabled in Carbon, only the Log4j appenders will write to the log files. If the Java Logging Handlers have logs, these logs will be delegated to the log events of the corresponding Log4j appenders. A Pub/Sub registry pattern implementation has been used in the latter mentioned scenario to plug the handlers and appenders. 
```

Comment:
@dinusha92 , is it ok to mention Hazelcast here in the note and in the configurations?

Reply:
@Mariangela this has been there for a time. Actually I am not aware why this has been added initially.

Reply:
@ruks , please comment whether it is correct to state Hazelcast here